File: 2.0/java/keywords/updateKeywordLease.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.keywords.model.KeywordLease;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        KeywordLease keywordLease = new KeywordLease();
        keywordLease.setKeyword("SUN");
        keywordLease.setAutoRenew(false);
        client.keywordLeasesApi().update(keywordLease);
    }
}

File: 2.0/csharp/keywords/updateKeywordLease.cs
using CallfireApiClient;
using CallfireApiClient.Api.Keywords.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var lease = new KeywordLease
        {
            KeywordName = "CALLFIRE",
            AutoRenew = false
        };
        client.KeywordLeasesApi.Update(lease);
    }
}
